### ISSUE TITLE
Feature Request: Support popup window

### ISSUE THREAD

**Issue by esambo** [dexterleng/vimac-releases#12]
I cannot use Vimac for popup windows (Accessibility Inspector.app _Hierarchy_ shows it as: `(floating window) [NSPanel]`), as nothing is recognized

**Comment by esambo** [comment]
It now works, after I restarted my computer...
Is there a better way to troubleshoot besides restarting? I wish there were a verbose log file...
